app/src/main/java/com/example/poste/activities/EditPostActivity.java

```diff
@@ -71,6 +72,16 @@ protected void onCreate(Bundle savedInstanceState) {
             postTitle.setText(currentPost.getTitle());
             postDescription.setText(currentPost.getDescription());
             postLink.setText(currentPost.getUrl());
+            if (!currentPost.getTags().isEmpty()) {
```

Comment:
I think this can actually be dropped; if we're just iterating over the tags, an empty tag list will just end right away

Reply:
That's some AM-hours coding side affects there.
Removed.